### PR TITLE
[SCB-2533] [master] encode invocation context before add to http header

### PR DESCRIPTION
**Pull request by demonbug**
fix issus https://github.com/apache/servicecomb-java-chassis/issues/3008

add writeUnicodeValueAsString to JsonUtils to support unicode encoding
encode invocation context before add to http header

### CODE REVIEW

File: dependencies/default/pom.xml [hunk 1]
```diff
@@ -44,6 +44,7 @@
     <commons-io.version>2.11.0</commons-io.version>
     <commons-lang.version>2.6</commons-lang.version>
     <commons-lang3.version>3.12.0</commons-lang3.version>
+    <commons-text.version>1.9</commons-text.version>
```
Comment (Shoothzj) [inline dependencies/default/pom.xml:47]: we sort dependencies by alphabeta, it should below `commons-logging`

**Comment by Shoothzj** [comment]
LGTM, one nit.

**Review by Shoothzj — APPROVED** [review]

File: dependencies/default/pom.xml (hunk 1 above)
Comment (demonbug) [reply dependencies/default/pom.xml]: fixed

**Comment by Shoothzj** [comment]
Thanks for your contribution. And welcom to servicecomb community.

**Pull request merged**